app/serializers/question_serializer.rb

```diff
@@ -1,8 +1,28 @@
 class QuestionSerializer < ActiveModel::Serializer
   attributes :id, :user_id, :title, :content, :votes, :comments, :tags, :answers,
-             :created_at, :updated_at
+             :created_at, :updated_at, :votes_count, :answers_count, :views_count,
+             :time_updated, :user
 
-   has_many :comments, as: :comment_on
-   has_many :tags, as: :subscriber
-   has_many :answers
+  has_many :comments, as: :comment_on
+  has_many :tags, as: :subscriber
+  has_many :answers
+
+  def votes_count
+    object.votes.count
```

Comment:
The `votes` are stored as either 1 or -1in the `value` column. What you may want is the sum(:value) not count. It's been fixed in my `PR`, though